src/network/networkprotocol.h

```diff
@@ -229,6 +229,12 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 		[bump for 5.9.1]
 	PROTOCOL VERSION 46:
 		Move default hotbar from client-side C++ to server-side builtin Lua
+		Move death screen to server and make it a regular formspec
+			The server no longer triggers the hardcoded client-side death
+			formspec, but the client still supports it for compatibility with
+			old servers.
+		Rename TOCLIENT_DEATHSCREEN to TOCLIENT_DEATHSCREEN_LEGACY
+		Rename	TOSERVER_RESPAWN to TOSERVER_RESPAWN_LEGACY
 		[scheduled bump for 5.10.0]
```

Comment:
as there is a scheduled bump for 5.10 planed, shouldn't we just drop those legacy ?

Reply:
"bump" means we increase the protocol version, not that we remove backwards-compatibility

Reply:
@sfan5 ah yeah right it's not the min which is increased :)